dnadoh/dnadoh_plot/plot_util.py

```diff
@@ -0,0 +1,195 @@
+"""Plotting utils"""
+
+import plotly.io as pio
+import plotly.express as px
+import plotly.graph_objects as go
+import pandas as pd
+import numpy as np
+from plotly.subplots import make_subplots
+from scipy import stats
+
+# this file serves to keep all the plotting functionality in one place
+# the idea is to separate the figure creation process from the dash framework
+
+
+def get_unique_locations(df: pd.DataFrame) -> list:
+    """Get unique locations with an alternate base"""
+    return df["location"].drop_duplicates().dropna().to_list()
```

Comment:
I think you can just use `df["location"].unique()` here.

Reply:
Although I think yours works too!

Reply:
this is definitely from my convoluted code 😂 yeah I think Brodie's works better (though would want to confirm if NA's or dropped or if we still need that? 
also i think I wanted this in a list - does `df["location"].unique()`return a series or list?